core/src/avm2/globals/flash/display/ScreenMode.as

```diff
@@ -0,0 +1,41 @@
+package flash.display
+{
+  import __ruffle__.stub_getter;
+  import flash.geom.Rectangle;
+
+  // [API("733")] Ruffle doesn't support this API Version
+  [API(661)]
```

Comment:
Can this put the versioning in quotes like `[API("661")]` for consistency with other versioning metadata?